### DISCUSSION TITLE
Le monde des courbes entre complétude des écritures et incomplétudes des productions et des problèmes

### DISCUSSION THREAD

**Discussion by johannphilippe** [johannphilippe/hypercurve#3] [Announcements]

**Comment by Francois-Roux** [comment]
Pour contribuer à comprendre pourquoi la question des courbes est un élément central, je tombe ce matin sur l'un des séminaires mamuphi-IRCAM très pédagogique dont la référence est : Évelyne Barbin : Le monde des courbes entre complétude des écritures et incomplétudes des productions et des problèmes. Du 
16 mars 2019
http://www.entretemps.asso.fr/2018-2019/index.html#_edn11

dont voici l'abstract :
"L’histoire des courbes peut être lue comme une succession de moments où la conception d’un monde complet de courbes éclate par raison d’incomplétude pour se refermer sur une nouvelle complétude. Nous proposons d’examiner quatre de ces moments pour montrer l’importance jouée par la nature licite ou non de produire des courbes et par la résolution de problèmes, que l’on se doit de résoudre ou pas..."

**Comment by johannphilippe** [comment]
Ce qui nous invite à penser que nous n'utilisons pas le mot courbe au sens mathématique du terme, et qu'on devrait y privilégier le mot fonction ? Hypercurve est alors plutôt un outil d'assemblage de fonctions par morceaux. 
Pour le reste, très intéressant, pas eu le temps de tout regarder mais je m'y met bientôt. Merci !
